packages/components/src/highlight-cards/annual-highlight-cards.tsx

```diff
@@ -58,6 +57,18 @@ function AnnualHighlightsStandard( { counts }: AnnualHighlightsProps ) {
 	);
 }
 
+function HighlightCardsHeading( { children }: { children: React.ReactNode } ) {
```

Comment:
What's the rubric we're using to determine when to consolidate components into a file, vs. splitting out to a separate file? Since we're doing a bit of both in this pull request with `HighlightCardsHeading` and `TooltipContent`.

I'm also coming from a personal convention of strict one-component-per-file 😄

Reply:
Usually, a component per file makes sense. This just felt too small to be its own file 😉 Can totally be separate though, I wouldn't be opposed.